source/usage-examples/findOneAndUpdate.txt

```diff
@@ -0,0 +1,42 @@
+============================
+Update a Document Atomically
+============================
+
+.. default-domain:: mongodb
+
+A single document can be updated and returned in one single atomic
```

Comment:
I would recommend passing over this whole document with active/passive voice in mind. Especially in an opener I'd probably start with "You can update and return a single document in an atomic operation with..." and then elaborate about what all of those concepts mean.